iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/openssl.dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:22.04
 
 # Noncompliant@+1 {{Revoke and change this secret, as it might be compromised.}}
 RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048
-#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I would add below test with highlighting :
```
RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048 && other command
RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048 | other command
RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048; other command
```

Reply:
In this case the other command would also be flagged.